dependencies.yaml

```
@@ -164,7 +164,7 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - &cmake_ver cmake>=3.26.4,!=3.30.0
-          - cython>=3.0.0
+          - cython>=3.0.0,<=3.1.0a0
```

Is this supposed to be strictly less than?
```suggestion
          - cython>=3.0.0,<3.1.0a0
```

---

Ahhh yes!!! Thank you for catching this 🙈 

Updated all of them in https://github.com/rapidsai/raft/pull/2490/commits/42064859d662c67bf342f6b79633e63a52c66ce2